examples/src/pages/e2e/pointcloud/ColorStyling.tsx

```diff
@@ -0,0 +1,54 @@
+/*!
+ * Copyright 2022 Cognite AS
+ */
+
+import React from 'react';
+import { registerVisualTest } from '../../../visual_tests';
+
+import {
+  Cognite3DViewerOptions,
+  PotreePointColorType,
+  AnnotationListStylableObjectCollection,
+  CognitePointCloudModel
+} from '@cognite/reveal';
+import { Cognite3DTestViewer } from '../Cognite3DTestViewer';
+import { CogniteClientPlayground } from '@cognite/sdk-playground';
```

Comment:
Is Playground SDK still necessary?

Reply:
Yes, will fix in a later PR

Reply:
On second thought, I just pushed a commit where it is removed